### PR TITLE
Add the ability to opt in and out of weekly emails

### LINKED ISSUE #7154: Create button to toggle weekly email preferences
I will do this live on Twitch tonight.

### PR DESCRIPTION
**Pull request by QuincyLarson**
closes #7154

### CODE REVIEW

File: server/boot/user.js [hunk 1]
```diff
@@ -424,6 +429,37 @@ module.exports = function(app) {
...
+            You can turn these back on any time in the "Manage My Account"
+            section at the bottom of this page.
+          `
+        });
+        res.redirect('/' + req.user.username);
+      });
+    }
+    req.user.sendMonthlyEmail = true;
+    return req.user.save(function(err) {
+      if (err) { return next(err); }
```
Comment (BerkeleyTrue) [inline server/boot/user.js]: This should be done with and update instead of a save.

Comment (Cade9) [inline server/boot/user.js]: Ok

File: server/boot/user.js [hunk 2]
```diff
@@ -424,6 +429,37 @@ module.exports = function(app) {
...
+
+      req.flash('success', {
```
Comment (Game21) [inline server/boot/user.js]: Yes

File: server/boot/user.js (hunk 1 above)
Comment (Game21) [inline server/boot/user.js]: Update

**Pull request closed without merging**